src/components/BrowserSupportTable/styles.module.css

```diff
@@ -0,0 +1,79 @@
+th.rotate {
+  height: 180px;
+  white-space: nowrap;
+  vertical-align: bottom;
+}
+
+th.rotate > span > div {
+  transform: rotate(270deg);
+  width: 30px;
+}
+th.rotate > span > div > span {
+  padding: 5px 10px;
+}
+
+.tableWidth {
+  max-width: max-content;
+  margin: auto;
+}
+
+.myTest {
```

Comment:
Are you using this css? If not, we can remove it 👍

Reply:
Sure, thanks